UN-OCHA/unocha-site
# [UNO-786] Use cd-disclosure for OCHA donors

Refs: UNO-786

This replaces the use of the `cd-toggable` for the OCHA top donors with the `cd-disclosure` introduced in CD 9.2

**Note:** There is a slight difference with the cd-toggable version visible on https://www.unocha.org/our-funding. There is no transition on the chevron icon. That's a limitation of the `cd-disclosure` component where we cannot specify an icon (which was a `svg` in the button) and we have to relay on the `rw-icons` component to display it. The `rw-icons` uses a svg sprite so we cannot have a transition on the background color. It's a really minor thing so can be ignored.

### Tests

1. Checkout the branch, run composer install or recreate the image, clear the cache.
2. Visit local `/our-funding`, check that the list of donors shows only the first 10 ones
3. Check that there is a `Show all donors` button
4. Click the button and check that now the full list of donors is displayed.
5. Scroll, click the button to switch back to showing only the first 10 ones.
6. Confirm that it's the same behavior as https://www.unocha.org/our-funding
